eea/coremetadata/profiles/default/metadata.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <metadata>
-  <version>4.0</version>
+  <version>5.1</version>
```

Comment:
@laszlocseh Why 5.1? It should be `4.4`, the next egg version. Or if you think it's a major, we can bump to `5.0`. But I don't see why it's `5.1`